programs/lfx-mentorship/2024/01-Mar-May/README.md

```diff
@@ -28,8 +28,95 @@ Mentee application instructions can be found on the [Program Guidelines](https:/
 ---
 
 ## Table of Contents
+- [Term 01 - 2024 March - May](#term-01---2024-march---may)
+    - [Timeline](#timeline)
+    - [Project Instructions](#project-instructions)
+    - [Application instructions](#application-instructions)
+  - [Table of Contents](#table-of-contents)
+  - [Accepted Projects](#accepted-projects)
```

Comment:
Could we drop this section? All we need for the ToC here is the list of projects (everything else is above the ToC so should be fairly easy to find 🙂)

Reply:
Alright 👍